Taskfile.yml

```diff
@@ -105,6 +108,22 @@ tasks:
       - task: deps:default
       - lefthook install
 
+  portman_tests:
+    desc: "Run the OpenAPI specification tests from schema tests"
+    dotenv: [ 'ci/tests/schema/specs/.env' ]
+    cmds:
+      - task portman_integration_tests:portman_tests
+
+  run_gateway_to_test_oas:
+    desc: "Start the gateway to test portman"
+    dotenv: [ 'ci/tests/schema/specs/.env' ]
+    cmds:
+      - task portman_integration_tests:run_gateway_to_test_oas
+  stop_gateway_to_test_oas:
+    desc: "Stop the gateway to test portman"
+    dotenv: [ 'ci/tests/schema/specs/.env' ]
+    cmds:
+      - task portman_integration_tests:stop_gateway_to_test_oas
```

Comment:
Naming/structure could do with a little bit of work, sort of like .taskfiles/test, lint, hooks (example).
You could just import it with shorter aliases ("schema"), and move all the added tasks into the specs taskfile.

`task schema:test`; also other taskfiles show how to use `services:up` (/tests/**/Taskfile.yml); If you have additional services or your own docker-compose file, certain patterns should be kept.

Also, do they need to be in the root taskfile at all?
See: https://github.com/TykTechnologies/tyk/pull/6851/files for inspiration.

Reply:
`task -l` lists imported taskfiles, opentelemetry is an example of the required change (just import). Move all other logic (lifecycle or otherwise) into the ci/tests/schema folder where you import taskfile from.

the idea is that i go into that folder and run `task -l` and run the tests from there, if I'm doing any kind of manual debugging that's my test scope, my test scope shouldn't include anything other than `/docker` (test services), and could be `chroot`-ed in place. Also makes them moveable out of repo.